imgee/storage.py

```diff
@@ -77,18 +103,19 @@ def save_img_in_db(name, title, local_path, profile, mimetype, orig_extn):
     return stored_file
 
 
-def save_tn_in_db(img, tn_name, (tn_w, tn_h)):
+def save_tn_in_db(img, tn_name, size):
     """
     Save thumbnail info in db.
     """
+    tn_w, tn_h = size
     name, extn = os.path.splitext(tn_name)
-    tn = Thumbnail(name=name, width=tn_w, height=tn_h, stored_file=img)
-    db.session.add(tn)
-    db.session.commit()
+    if Thumbnail.query.filter(Thumbnail.name == name).isempty():
```

Comment:
What are these `name` fields here? On the Thumbnail model and in the `tn_name` parameter? Please add comments to clarify. This may also need a separate ticket to determine if we're doing this correctly. For example, it should be possible to make both JPEG and WebP thumbnails of the same image.